pkg/plugins/golang/v3/commons.go

```diff
@@ -78,7 +78,7 @@ manifests: controller-gen`
 		}
 
 		if err := util.ReplaceInFile("Makefile",
-			"ENVTEST_K8S_VERSION = 1.23",
+			"ENVTEST_K8S_VERSION = 1.23.5",
```

Comment:
@NikhilSharmaWe sorry, this one we will only update when we update the plugin scaffolds 
This implementation is because we have a feature to still use v1beta1 CRD APIs and it is removed from the latest versions so that if someone uses the feature we need to replace the scaffold to try to downgrade it all.

Reply:
ok